src/config.ts

```diff
@@ -323,7 +323,7 @@ export default {
    */
   DEFAULT_LIVE_GAP: {
     DEFAULT: 10,
-    LOW_LATENCY: 3,
+    LOW_LATENCY: 3.5,
```

Comment:
Sadly, I decided to add 500 milliseconds to the default starting position because it has an impact on the adaptive algorithm chosen here.